flow360/component/simulation/simulation_params.py

```diff
@@ -111,7 +111,7 @@ def _init_with_context(self, **kwargs):
         Initializes the simulation parameters with the given unit context.
         """
         kwargs = self._init_check_unit_system(**kwargs)
-        super().__init__(unit_system=unit_system_manager.copy_current(), **kwargs)
+        super().__init__(unit_system=unit_system_manager.current, **kwargs)
```

Comment:
are we sure it will not overwrite unit system when two SimulationParams will be created in different unit systems one after the other?

Reply:
Yes, I added a unit test.